eng/pipelines/aggregate-reports.yml

```diff
@@ -17,144 +22,166 @@ pr:
+            inputs:
+              sourcePath: '$(Build.ArtifactStagingDirectory)/reports'
+              azureSubscription: 'Azure SDK Artifacts'
+              destination: AzureBlob
+              storage: azuresdkartifacts
+              containerName: 'azure-sdk-for-net'
+              blobPrefix: dependencies
+          - task: PowerShell@2
+            displayName: "Verify Repository Resource Refs"
+            inputs:
+              pwsh: true
+              workingDirectory: $(Build.SourcesDirectory)
+              filePath: eng/common/scripts/Verify-Resource-Ref.ps1
+          - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@3
+            displayName: 'Run CredScan'
+            inputs:
+              suppressionsFile: 'eng\CredScanSuppression.json'
+            condition: succeededOrFailed()
+          - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
+            displayName: 'Post Analysis'
+            inputs:
+              GdnBreakAllTools: false
+              GdnBreakGdnToolCredScan: true
+              GdnBreakGdnToolCredScanSeverity: Error
+              GdnBreakBaselineFiles: $(Build.SourcesDirectory)\eng\dotnet.gdnbaselines
+              GdnBreakBaselines: baseline  
+              # Used for generating baseline file
+              # GdnBreakOutputBaselineFile: dotnet
+              # GdnBreakOutputBaseline: baseline
+            condition: succeededOrFailed()
+            continueOnError: true
+          - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
+            displayName: 'Publish Security Analysis Logs'
+            condition: succeededOrFailed()
+
+  - stage: PoliCheck
+    displayName: PoliCheck Evaluation
+    dependsOn: []
+    condition: ne(variables['ServiceDirectoryName'], '')
+    jobs:
+      - job: PoliCheckRunAndPublish
+        timeoutInMinutes: 120
+        steps:
+          - checkout: self
+          - checkout: azure-sdk-build-tools
+            fetchDepth: 1
+          - task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@2
+            displayName: 'Run PoliCheck'
+            inputs:
+              targetType: F
+              targetArgument: '$(Agent.BuildDirectory)/s/azure-sdk-for-net/sdk/$(ServiceDirectoryName)'
+              result: PoliCheck.sarif
+              optionsFC: 0
+              optionsXS: 1
+          - task: PowerShell@2
+            displayName: 'Filter PoliCheck Results using AllowList'
+            inputs:
+              pwsh: true
+              filePath: '$(Agent.BuildDirectory)/s/azure-sdk-for-net/eng/common/scripts/FilterPoliCheckResults.ps1'
+              workingDirectory: $(Pipeline.Workspace)
+              arguments: >
+                -PoliCheckResultFilePath '$(Agent.BuildDirectory)/.gdn/.r/PoliCheck/001/PoliCheck.sarif'
```

Comment:
What populates this directory? The policheck task?

Reply:
Right.